src/realm/bplustree.cpp

```diff
@@ -144,7 +144,8 @@ class BPlusTreeInner : public BPlusTreeNode, private Array {
         REALM_ASSERT_DEBUG(!m_offsets.is_attached());
         return size_t(Array::get(0)) >> 1;
     }
-    ref_type get_child_ref(size_t ndx) const noexcept override
+    // Should not be mixed up with Array::get_child_ref
+    ref_type _get_child_ref(size_t ndx) const noexcept
```

Comment:
what is the real meaningful difference? Perhaps better naming is possible?

Reply:
The only difference is that _get_child_ref() add one to ndx before getting the value. I could not come up with a better name. Open for suggestions.